parlai/agents/seq2seq/hred.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/env python3
```

Comment:
how do you feel about making this its own agent?

parlai/agents/hred/ ? I think it would be better, and highlight it as its own thing.

additionally, separate the HredAgent into hred.py, and put all the pytorch modules into a `modules.py`.

Reply:
Can you also add an integration test for this? Something like these two:

https://github.com/facebookresearch/ParlAI/blob/4b257b8509a309870f1037b61fbf268797443741/tests/test_transformers.py#L252-L304

Additionally, maybe we can try to do a pretrained model for `-t integration_tests:multiturn_candidate`. Something with like 1 layer, 16 esz, 32 hidden size

Reply:
Yup I can definitely make it its own agent and add the integration test--how do I get the baseline valid['metric'] figures? Just run it myself?